.travis.yml

```diff
@@ -1,23 +1,20 @@
 language: cpp
-cache: ccache
+cache:
+  ccache: true
+  directories:
+    - CachedCMakeCache # Avoid rediscovery of types sizes
+    - Testing          # Store ctest cost data
```

Comment:
What does this do?

Reply:
Admittedly small benefit, where ctest can keep track of how long a test took ("the cost") and optimise their run order the next time around. So LuaStressTest (10 seconds) gets run first, instead of the order it was declared (close to last) and holding one thread hostage when everything else is done.